src/plugins/intel_gpu/src/kernel_selector/cl_kernels/dynamic_quantize_gpu_ref.cl

```diff
@@ -8,6 +8,20 @@
 #error "dynamic_quantize_gpu_ref.cl: Unsupported output dimension"
 #endif
 
+#if VEC_SIZE == 1
+    #define VSTORE_N vstore
+    #define HALF_N half
+    #define AS_HALF_TYPE_N(x) (half)
+#else
+    #define VLOAD_N CAT(vload, VEC_SIZE)
+    #define VSTORE_N CAT(vstore, VEC_SIZE)
+    #define HALF_N CAT(half, VEC_SIZE)
+    #define CONVERT_CHAR_N CAT(convert_char, VEC_SIZE)
+    #define AS_TYPE_N_(type, n, x) as_##type##n(x)
+    #define AS_TYPE_N(type, n, x) AS_TYPE_N_(half, n, x)
+    #define AS_HALF_TYPE_N(x) AS_TYPE_N(half, VEC_SIZE, x)
+#endif
```

Comment:
do we need these changes? did original implementation have issue with unaligned case?

Reply:
original code just does vload8 without checking size.

Reply:
It calls vload8 with INPUT0_SIZE_X / 8. Isn't it sufficient?